content/_data/logo/santa-claus.yml

```
@@ -1,5 +1,5 @@
 ---
-name: 'Santa Claus'
+name: 'Santa Clause'
```

Name of a person (albeit mythical)
```suggestion
name: 'Santa Claus'
```